src/app/modules/bounce/bounce.service.ts

```diff
@@ -129,18 +137,34 @@ const logCriticalBounce = (
 export const updateBounces = async (body: ISnsNotification): Promise<void> => {
   const notification: IEmailNotification = JSON.parse(body.Message)
   // This is the crucial log statement which allows us to debug bounce-related
-  // issues, as it logs all the details about deliveries and bounces
-  logger.info(notification)
+  // issues, as it logs all the details about deliveries and bounces. Email
+  // confirmation info goes to the short-term log group so we do not store
+  // form fillers' information for too long, and everything else goes into the
+  // main log group.
+  if (
+    extractHeader(notification, EMAIL_HEADERS.emailType) ===
+    EMAIL_TYPES.emailConfirmation
+  ) {
+    shortTermLogger.info(notification)
+  } else {
+    logger.info({
+      message: 'Email notification',
+      meta: {
+        action: 'Received notification from SNS',
```

Comment:
Same here, action should be the method invocation name, in this case `updateBounces`.